azext_iot/central/params.py

```diff
@@ -155,10 +160,17 @@ def load_central_arguments(self, _):
         context.argument(
             "device_id",
             options_list=["--device-id", "-d"],
-            help="Provide a unique identifier for the device."
+            help="Unique identifier for the device."
             " A case-sensitive string (up to 128 characters long) of ASCII 7-bit alphanumeric characters plus"
             " certain special characters: - . + % _ # * ? ! ( ) , : = @ $ '",
         )
+        context.argument(
+            "organizations",
+            options_list=["--organizations", "--org"],
```

Comment:
`--organizations` is plural but `--org` is not?

Reply:
fixed